clients/itwin/src/Token.ts

```diff
@@ -46,151 +51,7 @@ class TokenPrefixToTypeContainer {
  */
 export interface AccessTokenProps {

```

Comment:
Where are all of the types in this file used? I'm hoping we'd be able to get rid of most if not all of them.

Reply:
Yeah I think we can get rid of the whole accessTokenProps. The way the code is now it's only really using the `tokenString` in it.